### ISSUE TITLE
Fix pdf downloads blocked on some urls

### ISSUE THREAD

**Issue by Phiphigengen** [dataforgoodfr/12_taxobservatory#70]

**Comment by FS-CS** [comment]
https://github.com/dataforgoodfr/12_taxobservatory/issues/50#issuecomment-2063265611

**Comment by Phiphigengen** [comment]
Closed because duplicate of https://github.com/dataforgoodfr/12_taxobservatory/pull/72
